examples/ensemble/plot_stack_predictors.py

```diff
@@ -16,42 +16,127 @@
 print(__doc__)
 
 # Authors: Guillaume Lemaitre <g.lemaitre58@gmail.com>
+#          Maria Telenczuk    <https://github.com/maikia>
 # License: BSD 3 clause
 
+
 ###############################################################################
-# The function ``plot_regression_results`` is used to plot the predicted and
-# true targets.
+# Download the dataset
+###############################################################################
+#
+# We will use `Ames Housing`_ dataset which was first compiled by
+# Dean De Cock and became better known after it was used in `Kaggle
+# challenge`_. It is a set of 1460 residential homes in Ames, Iowa, each
+# described by 80 features. The task is to predict the final price of the
+# houses. In this example we will use only 20 most intersting features chosen
+# using GradientBoostingRegressor() and limit number of entries to 750 (here we don't show how to select most
```

Comment:
This is causing the linting CI to fail:

https://app.circleci.com/jobs/github/maikia/scikit-learn/43

The end of the paragraph should be reformatted to fit in less than 80 chars.